### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/961
https://github.com/open-orchestra/open-orchestra-model-interface/pull/98
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/386
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/111
https://github.com/open-orchestra/open-orchestra-front-demo/pull/67
https://github.com/open-orchestra/open-orchestra-provision/pull/15
https://github.com/open-orchestra/open-orchestra-docs/pull/66

### CODE REVIEW

File: en/developer_guide/error_pages.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    web/*siteId*/alias-*aliasId*/errorPage503.html
+
+where *siteId* is the site id and *aliasId* the alias id.
+
+Configuring the vhost
+~~~~~~~~~~~~~~~~~~~~~
+To use the 503 static page, the apache vhost of the site must be configured using the ErrorDocument 
+directive. Here comes a version of the configuration allowing you to avoid the vhost file edition
+each time you want to put on/off your site.
```
Comment (juchi) [inline en/developer_guide/error_pages.rst]: "Virtual Host" is more suited when writing than "vhost"

File: en/developer_guide/error_pages.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To contribute the 503 error page, in the Back Office navigation menu, click on "Error pages", then
+on "503". This will show a standard node configuration page. The node can be contributed for each
+available language of the Front Office. To make the page available, it must be published.
+
+Dump the node into statics files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Open Orchestra comes with a command line to dump the error nodes into static files. You can execute
+that command line at the root of your project or place the command in a cron task to dump the files
+periodicly.
```
Comment (juchi) [inline en/developer_guide/error_pages.rst]: periodically

File: en/developer_guide/error_pages.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+The command is the following:
+
+    app/console orchestra:errorpages:generate [siteId]
+
+The siteId is an option allowing you to specify a specific site to limit the dump operation. Without
+that option the pages are dumped for every sites.
```
Comment (juchi) [inline en/developer_guide/error_pages.rst]: every site

File: en/developer_guide/error_pages.rst [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+For each site, the command will look for error nodes and especially the published versions. The script
+will then look to the available site alias. When an error node is published in the site alias language,
```
Comment (juchi) [inline en/developer_guide/error_pages.rst]: site aliases ?

File: en/developer_guide/error_pages.rst [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+the script generates the matching files.
+For instance if mysite.com declares an alias in english and a second one in french, and if a 503 error
+nodes is published in english, then the command generates a 503 file in english for the first alias.
```
Comment (juchi) [inline en/developer_guide/error_pages.rst]: node

File: en/developer_guide/error_pages.rst [hunk 6]
```diff
@@ -0,0 +1,75 @@
+Error pages
+===========
+
+Symfony allows a developper to customize error pages shown by the framework when a matching http
```
Comment (juchi) [inline en/developer_guide/error_pages.rst]: HTTP should be in uppercase

**Comment by ngilain** [comment]
@juchi : update done

**Comment by juchi** [comment]
good for me

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**